src/styles/inputs.js

```diff
@@ -0,0 +1,17 @@
+import { css } from 'styled-components';
+import theme from 'styles/theme';
+
+const inputBase = css`
+  border: 1px solid ${theme.color.light_grey};
+  border-radius: 4px;
+  color: ${theme.color.dark_gray};
+  display: table;
+  font: ${theme.fontSize.base} ${theme.fontFamily};
+  height: 36px;
+  line-height: 34px;
```

Comment:
Line-height usually should not be in px, should be unit-less. Is there a reason to have px here?

Reply:
I think I was just matching the styles of the text in the Select inputs. Might have been a calculated property I grabbed on accident? Probably not even needed.